distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -49,16 +49,21 @@ javax.net.ssl.trustStoreType=jks
 #
 # Global URL Properties
 #
-# The httpsPort, httpPort, and port properties are used to advertise the system externally
-# The internalHttpsPort and internalHttpPort properties are the ports that the system will run on
+# The httpsPort and httpPort are the ports that the system will run on
 org.codice.ddf.system.protocol=https://
 org.codice.ddf.system.hostname=localhost
 org.codice.ddf.system.httpsPort=8993
 org.codice.ddf.system.httpPort=8181
 org.codice.ddf.system.port=_DO_NOT_EXPAND_${org.codice.ddf.system.httpsPort}
-org.codice.ddf.system.internalHttpsPort=_DO_NOT_EXPAND_${org.codice.ddf.system.httpsPort}
-org.codice.ddf.system.internalHttpPort=_DO_NOT_EXPAND_${org.codice.ddf.system.httpPort}
 org.codice.ddf.system.rootContext=/services
+# external.httpsPort and external.httpPort are useful when DDF is behind a proxy and
+# the URL a user sees is different than the local url DDF uses
+org.codice.ddf.external.protocol=https://
+org.codice.ddf.external.hostname=localhost
+org.codice.ddf.external.httpsPort=8993
```

Comment:
Is there a reason we decided to go with external here as opposed to expanding on the pre-existing internal properties? I think most cases care about the external and only a handful would care about the internal. This could make for a much smaller changes set and maybe less confusion in the future.